src/main/java/duke/tasks/Deadlines.java

```diff
@@ -0,0 +1,41 @@
+package duke.tasks;
+
+import java.time.LocalDateTime;
+
+/**
+ * This class represents a task with a deadline.
+ */
+public class Deadlines extends Task {
```

Comment:
Perhaps this class and all other classes that inherit from Task should be phrased in singular form?
```suggestion
public class Deadline extends Task {
```